src/Test/Perf/Runner/Perf.Runner.csproj

```diff
@@ -80,4 +80,10 @@
     <Folder Include="CPC-Consumption\" />
   </ItemGroup>
   <Import Project="..\..\..\..\build\Targets\VSL.Imports.targets" />
+  <PropertyGroup>
+    <PostBuildEvent>call "$(MSBuildThisFileDirectory)install_benchview_tools.cmd" "$(OutDir)"</PostBuildEvent>
```

Comment:
I'm looking into how best to achieve this. I understand why you'd prefer not to have it as a post-build step here. 

The difference between this and the other nuget dependencies is that this package doesn't include any .NET assemblies that are referenced, but we need to get some files from inside the package into the build directory, along side the outputted Perf.Runner.exe. That is why we invoke `nuget install` here directly and provide a output directory, instead of declaring this as a dependency of the Perf.Runner.csproj.

I'm thinking about moving this to be called from ./build/scripts/run_perf.ps1 (which downloads CPC). What do you think about that approach?